### PR TITLE
Makes the announcement banner reusable with message and event props

### PR DESCRIPTION
**Pull request by dakota002**
How it would look:

image

How to use:

image

### CODE REVIEW

File: app/components/AnnounceBanner.tsx [hunk 1]
```diff
@@ -46,30 +55,19 @@ export default function AnnounceBanner() {
...
+              <div className="grid-row">
+                {events.map((event) => (
+                  <div key={event.link} className="mobile-lg:grid-col-4">
+                    {event.time}
+                    <div>(best for {event.region}):</div>
+                    <Link rel="external" href={event.link}>
+                      {event.link}
+                    </Link>
+                  </div>
+                ))}
```
Comment (lpsinger) [inline app/components/AnnounceBanner.tsx:68]: It would be simpler to extract this as another component.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/components/AnnounceBanner.tsx (hunk 1 above)
Comment (dakota002) [reply app/components/AnnounceBanner.tsx:68]: Will do!

**Comment by dakota002** [comment]
Revised usage example:
image

File: app/components/AnnounceBanner.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default function AnnounceBanner({
   events,
 }: {
   message: string
-  events?: announcedEvents[]
+  events?: AnnouncedEvents[]
```
Comment (lpsinger) [inline app/components/AnnounceBanner.tsx]:
```suggestion
  children?: AnnouncedEvents[]
```

File: app/components/AnnounceBanner.tsx [hunk 3]
```diff
@@ -84,3 +78,15 @@ export default function AnnounceBanner({
     </section>
   )
 }
+
+export function AnnouncementEvent(event: AnnouncedEvents) {
```
Comment (lpsinger) [inline app/components/AnnounceBanner.tsx:76]:
```suggestion
export function AnnouncementEvent(props: AnnouncedEvents) {
```

Also just inline the type here in the function signature.

File: app/components/AnnounceBanner.tsx [hunk 4]
```diff
@@ -86,3 +66,19 @@ export default function AnnounceBanner() {
...
   )
 }
+
+export function AnnouncementEvent(props: {
+  time: string
+  link: string
+  region: 'Atlantic' | 'Pacific' | 'Asia and Oceania'
+}) {
+  return (
+    <div className="mobile-lg:grid-col-4">
```
Comment (lpsinger) [inline app/components/AnnounceBanner.tsx:76]:
```suggestion
    <Grid mobileLg={{col: 4}}>
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Pull request merged**